scripts/sct_maths.py

```diff
@@ -269,11 +275,11 @@ def main(args=None):
         data_out = sct_math.otsu(data, param)
 
     elif arguments.adap is not None:
-        param = convert_list_str(arguments.adap, "int")
+        param = arguments.adap
         data_out = sct_math.adap(data, param[0], param[1])
 
     elif arguments.otsu_median is not None:
-        param = convert_list_str(arguments.otsu_median, "int")
+        param = arguments.otsu_median
         data_out = sct_math.otsu_median(data, param[0], param[1])
```

Comment:
Splay is good for this:

```suggestion
        data_out = sct_math.otsu_median(data, *arguments.otsu_median)
```

But there should be a way to specify that `len(arguments.otsu_median) == 2` or bust. Maybe `list_type()` could be extended to take an size? There's details, like how to distinguish no size from zero size from upper size from specific size.

Reply:
I agree that this isn't ideal...

But, at that point, we're reimplementing built-in argparse functionality all over again (i.e. the combination of `nargs=2` and `type=int`) just to continue to preserve our custom delimiting. :stuck_out_tongue: